docs/parameters.rst

```diff
@@ -48,9 +48,9 @@ different behavior and some are supported out of the box:
 
 ``bool`` / :data:`click.BOOL`:
     A parameter that accepts boolean values.  This is automatically used
-    for boolean flags.  If used with string values ``1``, ``yes``, ``y``, ``t``
-    and ``true`` convert to `True` and ``0``, ``no``, ``n``, ``f`` and ``false``
-    convert to `False`.
+    for boolean flags.  If used with string values, ``1``, ``yes``, ``y``,
```

Comment:
```suggestion
    for boolean flags.  If used with the string values, ``1``, ``yes``, ``y``,
```

Reply:
All of your sugggestions combined are insufficient to produce a grammatical sentence.  It would have to be "If used with the string values *[no comma]* `1` ... and `on` *[insert comma]* the value is `True`, and *if used with the string values* `0` ... and `off` *[insert comma]* the value is `False`."  Moreover, the current sentence is perfectly grammatical, and I fail to see any improvements in what you're suggesting.

Reply:
What about going for something much simpler:

```rst
The following string values are converted to their respective boolean counterparts:
``1``, ``yes``, ``y``, ``t``, ``true``, ``0``, ``no``, ``n``, ``f`` and ``false``
```

I think it's completely obvious which ones become `true` and which ones `false`...